builders/manage/sudo/pause-transactions.md

```diff
@@ -0,0 +1,108 @@
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/pause-transactions/pause-transactions-1.webp)
+
+!!! warning
+    It's critical to protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Module and Transaction Names {: #modules-transaction-names }
+
+The `Tx-pause` module works by filtering the execution of certain transactions. To do so, it keeps an internal list of banned transactions identified by module and transaction name. This list is case-sensitive and works only when there is an exact match between one item in the paused transactions list and the transaction being processed. Therefore, using the exact names of the modules and the transactions is crucial.
+
+To find out the names of the modules available in your runtime, you need to read the `construct_runtime!()` section in the `lib.rs` file of your custom project. If your appchain is based on one of the official templates, refer to the [lib.rs](https://github.com/moondance-labs/tanssi/blob/master/container-chains/runtime-templates/frontier/src/lib.rs){target=\_blank} of the EVM template or the [lib.rs](https://github.com/moondance-labs/tanssi/blob/master/container-chains/runtime-templates/simple/src/lib.rs){target=\_blank} of the Substrate template.
+
+The following snippet is an example of how the `construct_runtime!()` section looks like. The module names are those located to the left of the colon.
+
+```rust
+construct_runtime!(
+    pub enum Runtime
+    {
+        ...
+        Migrations: pallet_migrations = 7,
+        MaintenanceMode: pallet_maintenance_mode = 8,
+        TxPause: pallet_tx_pause = 9,
+        Balances: pallet_balances = 10,
+        Multisig: pallet_multisig = 16,      
+        ...
+   }
+```
+
+To identify the transaction names included in a module, you need to refer to its source code. Modules built in [Substrate](/learn/framework/overview/#substrate-framework){target=\_blank} identify their transactions using a macro `#[pallet::call_index(INDEX)]`, where `INDEX` is a number. In the case of a [built-in module](/builders/build/customize/adding-built-in-module/){target=\_blank}, the code is located within the [FRAME folder](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame){target=\_blank} of the Polkadot-SDK repository. For example, if you want to know about the transaction names in the `Balances` module, refer to its [lib.rs](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/balances/src/lib.rs){target=\_blank} file and look for the function names below the `#[pallet::call_index(INDEX)]` macros. The following snippet is the transaction `transfer_allow_death` of the module `Balances`, which is the one used as an example in this guide:
```

Comment:
Should we use the EVM as an example? It is probably more famous?

We could also include a short list of some of the popular modules and methods? In a table?

Reply:
Yeah, evm is definitely more popular, but if we pause ethereum->transact, the example would essentially be a full stop, right? It wouldn't look much like a "controlled damage while leaving the rest untouched"

Reply:
Table for modules & methods, yeah! finding that is quite difficult for the users otherwise.
I'll add it to this article, but we'll move it later surely to its own page. (Maybe to this pallets documentation section we have in backlog)

Reply:
Ok sure let's keep it simple the  with balance transfer. I just worry that this might not mean a lot to actual devs